docs/style.md

```diff
@@ -2,45 +2,53 @@
 
 This is a set of recommended InSpec rules you should use when writing controls.
 
-## Control files
+# Control Files
 
-### 1. All controls should be located in the "controls" directory and end in ".rb"
+## Place control files in "controls/" and end them with ".rb"
```

Comment:
I think leaving the `in the "controls" directory` is clearer here then the forward slash.

Reply:
Adding the extra words causes it to wrap in an ugly way when rendered. :disappointed:

Reply:
Maybe make "controls/" a link to the directory in the repository so its a little more obvious?